session-name-1/slides/.gitignore

```diff
@@ -0,0 +1,2 @@
+/slides.pdf
+/_site
```

Comment:
Since all sessions will have pretty much the same `.gitignore`, wouldn't it make more sense to add these lines to the global `.gitignore` instead?

Reply:
I'm thinking of renaming the `slides.pdf` file into `<session-name>.pdf`. It will be easier to publish it like that.

Anyway, the PDF generation is still not working properly on my side, so I need to take another look.